source/_integrations/heatzy.markdown

```diff
@@ -0,0 +1,28 @@
+---
+title: Heatzy
+description: Instructions on how to integrate the Heatzy climate within Home Assistant.
+ha_category:
+  - Climate
+ha_domain: heatzy
+ha_release: 2022.12.0
+ha_codeowners:
+  - '@cyr-ius'
+ha_config_flow: true
+ha_platforms:
+  - climate
+  - switch
```

Comment:
The addition of new integration must be limited to a single platform. Additional platforms can be added in later follow up PRs.